app/api/events.py

```diff
@@ -98,7 +98,9 @@
     'social_links': fields.List(fields.Nested(SOCIAL_LINK), attribute='social_link'),
     'call_for_papers': fields.Nested(EVENT_CFS, allow_null=True),
     'version': fields.Nested(EVENT_VERSION),
-    'has_session_speakers': fields.Boolean(default=False)
+    'has_session_speakers': fields.Boolean(default=False),
+    'thumbnail' : fields.String(),
```

Comment:
Instead of `fields.String()` you could use `fields.Uri()` or `fields.ImageUri()` for proper data validation.